generators/app/templates/src/app/screens/Login/layout.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18next from 'i18next';
```

Comment:
import t using destructuring

Reply:
Mmm How ? I always use it like this

Reply:
```js
import { t } from 'i18next';
```

Reply:
I'll explain in case you don't understand why this works like this. i18next (the library) exports an object called `i18next`. That object has a `t` property. So, it's as if you had the following:
```
const i18next = { t: /* some value the library defines*/ };

// So you can do:
const t = i18next.t;
// or:
const { t } = i18next; // Like how we do with props sometimes
```